src/main/java/com/minecolonies/coremod/tileentities/TileEntityWareHouse.java

```diff
@@ -370,22 +372,25 @@ public NBTTagCompound writeToNBT(@NotNull final NBTTagCompound compound)
      */
     public void dumpInventoryIntoWareHouse(@NotNull final InventoryCitizen inventoryCitizen)
     {
-        for(int i = 0; i < inventoryCitizen.getSizeInventory(); i++)
+        for(int i = 0; i < inventoryCitizen.getSlots(); i++)
         {
             final ItemStack stack = inventoryCitizen.getStackInSlot(i);
 
             if(stack == null || stack.getItem() == null || stack.stackSize == 0)
             {
                 continue;
             }
-            @Nullable final TileEntityChest chest = searchRightChestForStack(stack);
-            if(chest == null)
+            @Nullable final IItemHandler handler = searchRightItemHandler(stack);
+            if(handler == null)
             {
                 LanguageHandler.sendPlayersMessage(getColony().getMessageEntityPlayers(), "com.minecolonies.coremod.wareHouse.full");
                 return;
             }
-            InventoryUtils.addItemStackToInventory(chest, stack);
-            inventoryCitizen.removeStackFromSlot(i);
+            final ItemStack extractedStack = inventoryCitizen.extractItem(i, stack.stackSize, true);
+            if (extractedStack != null) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)